feat(ContentContainer)!: update responsive padding (#1392)
Codecademy/client-modules: packages/styleguide/stories/Layouts/ContentContainer.stories.mdx

import { ContentContainer } from '@codecademy/gamut/src';
import title from '@codecademy/macros/lib/title.macro';
import { Canvas, Meta, Story } from '@storybook/addon-docs/dist/blocks';

import { PropsTable } from '~styleguide/blocks';

<Meta
  title={title}
  component={ContentContainer}
  args={{ children: 'Some random content yo' }}
  parameters={{
    subtitle:
      'A container element meant to hold all content on the page, and provide the correct horizontal padding.',
    source: 'gamut',
    figmaId: 'ZUJ7cfVgyoh1kiQYNN8Hbt',
  }}
/>

ContentContainer can be used when creating page layouts to contain content
within a maximum width, as well as centering that content. By default, it provides responsive padding according to the updated grid system (see figma link above).

Note: AppBar (used in Global Header) and Footer also use ContentContainer.

> Check with design team if further padding/margin is required on a page layout as additional spacing would cause it to be misaligned with the responsive padding set on Global Header and Footer.

Author: lmk if this all makes sense.. it might be more info than necessary 😅

Reviewer: This is the perfect amount of specificity, love it

> Check with design team if further padding/margin is required on a page layout as additional spacing would cause it to be misaligned with the responsive padding set on Global Header and Footer.

Reviewer: ```suggestion
Check with the design team if a different padding/margin is required on a page layout as additional spacing may be cause misalignment with the Header and Footer.
```
